pkg/cmd/pulumi/console.go

```diff
@@ -73,13 +73,13 @@ func newConsoleCmd() *cobra.Command {
 				// Open the stack specific URL (e.g. app.pulumi.com/{org}/{project}/{stack}) for this
 				// stack if a stack is selected and is a cloud stack, else open the cloud backend URL
 				// home page, e.g. app.pulumi.com.
-				if consoleURL, err := cloudBackend.StackConsoleURL(stack.Ref()); err != nil {
-					launchConsole(consoleURL)
-				} else {
```

Comment:
logic was flipped.

Reply:
I could make a 1 char replacement diff, but I think rewriting it would avoid the easy misreading of `==` in the first place.